### PR TITLE
support listArchives with sessionId, fixes #122

### LINKED ISSUE #122: OpenTok.listArchives() does not support filtering by session ID
The OpenTok REST API now lets you filter by session ID when getting a list of archives: https://tokbox.com/developer/rest/#listing_archives. (You pass in a `sessionId` query parameter when calling the REST API method.) The `OpenTok.listArchives()` methods in the OpenTok Java server SDK does not support this session ID filter.

### PR DESCRIPTION
**Pull request by kayhuba**
As @davideberlein already mentioned we're in need of this feature. Please review the PR and advise, thanks.

Regards
Kay

### CODE REVIEW

File: src/main/java/com/opentok/util/HttpClient.java [hunk 1]
```diff
@@ -133,6 +133,28 @@ public String getArchives(int offset, int count) throws RequestException {
...
     }
 
+    public String getArchives(String sessionId) throws RequestException {
+        String url = this.apiUrl + "/v2/project/" + this.apiKey + "/archive?sessionId=" + sessionId;
+
+        Future<Response> request = this.prepareGet(url).execute();
+        try {
+            Response response = request.get();
+            switch (response.getStatusCode()) {
+            case 200:
```
Comment (aiham) [inline src/main/java/com/opentok/util/HttpClient.java]: Could you indent everything in switch statement once?

Comment (aiham) [inline src/main/java/com/opentok/util/HttpClient.java]: Could you indent everything in switch statement once?

**Comment by kayhuba** [comment]
@aiham Thanks for the review! I've changed the base of the PR to dev now and fixed the indentation problem with the switch statement. Can you have another look?

**Comment by aiham** [comment]
Version 3.1.0 has been published. Thanks @kayhuba

**Comment by kayhuba** [comment]
Many thanks @aiham !

**Pull request merged**